cs/API-v2/Program.cs

```diff
@@ -131,7 +143,11 @@ static string sign(string stringToSign)
             }
         }
 
-        static string getApiKey()
+        static string GetUrlEncodedBody(Dictionary<String, String> param) {

```

Comment:
I don't get it, method name suggests it does encoding while in fact it does not :thinking: 
How about using `FormUrlEncodedContent` instead `StringContent` and if that's not possible due to signing then at least `HttpUtility.UrlEncode`?

Reply:
HttpUtility.UrlEncode does not do what we need:
It takes a string as an argument (`"Hello world and so on"`), and it returns a string, e.g. `"Hello+world+and+so+on".`

**FormUrlEncodedContent** works fine but cannot be used in this scenario either. That's because we need to have a string for signing. FormUrlEncodedContent is not a string, it's a buffer of some sort.

That's why I introduced this method, the name of which might be misleading. What about calling it `GetParamsAsString`?

@suside

Reply:
I see you went with `FormUrlEncodedContent` :+1: